Building from source fails on Fedora 33

For bug reports, the following information can help speed up the process. Please describe the bug that you have found and what you would expect to happen instead.

### Expected:

Install dependencies for Fedora and run `cargo build --release` results in no errors.

### Actual
Install dependencies for Fedora and run `cargo build --release` results in:

```
* Installing packages... 
   Compiling libc v0.2.79
   Compiling cfg-if v0.1.10
   Compiling pkg-config v0.3.19
   Compiling proc-macro2 v1.0.24
   Compiling log v0.4.11
   Compiling unicode-xid v0.2.1
   Compiling xml-rs v0.8.3
   Compiling bitflags v1.2.1
   Compiling lazy_static v1.4.0
   Compiling autocfg v1.0.1
   Compiling nix v0.18.0
   Compiling smallvec v1.4.2
   Compiling memchr v2.3.3
   Compiling cc v1.0.61
   Compiling version_check v0.9.2
   Compiling khronos_api v3.1.0
   Compiling syn v1.0.46
   Compiling byteorder v1.3.4
   Compiling getrandom v0.1.15
   Compiling once_cell v1.4.1
   Compiling downcast-rs v1.2.0
   Compiling scoped-tls v1.0.0
   Compiling ppv-lite86 v0.2.9
   Compiling same-file v1.0.6
   Compiling serde_derive v1.0.117
   Compiling ttf-parser v0.6.2
   Compiling serde v1.0.117
   Compiling ab_glyph_rasterizer v0.1.4
   Compiling siphasher v0.3.3
   Compiling slab v0.4.2
   Compiling xdg v2.2.0
   Compiling maybe-uninit v2.0.0
   Compiling scopeguard v1.1.0
   Compiling lazycell v1.3.0
   Compiling adler32 v1.2.0
   Compiling crc32fast v1.2.0
   Compiling ryu v1.0.5
   Compiling winit v0.23.0
   Compiling linked-hash-map v0.5.3
   Compiling arc-swap v0.4.7
   Compiling unicode-width v0.1.8
   Compiling serde_json v1.0.59
   Compiling dtoa v0.4.6
   Compiling utf8parse v0.2.0 (https://github.com/alacritty/vte?rev=4f44023dab081f7da74fee14bc53b10ee8f96a1e#4f44023d)
   Compiling fnv v1.0.7
   Compiling percent-encoding v2.1.0
   Compiling regex-syntax v0.6.20
   Compiling foreign-types-shared v0.3.0
   Compiling ansi_term v0.11.0
   Compiling itoa v0.4.6
   Compiling bytemuck v1.4.1
   Compiling vec_map v0.8.2
   Compiling strsim v0.8.0
   Compiling base64 v0.12.3
   Compiling urlocator v0.1.4
   Compiling libloading v0.6.4
   Compiling instant v0.1.7
   Compiling num-traits v0.2.12
   Compiling num-integer v0.1.43
   Compiling num-rational v0.3.0
   Compiling num-iter v0.1.41
   Compiling cmake v0.1.44
   Compiling wayland-sys v0.28.2
   Compiling x11-dl v2.18.5
   Compiling servo-fontconfig-sys v5.1.0
   Compiling crossfont v0.1.1
   Compiling nom v5.1.2
   Compiling walkdir v2.3.1
   Compiling phf_shared v0.8.0
   Compiling lock_api v0.4.1
   Compiling miniz_oxide v0.3.7
   Compiling owned_ttf_parser v0.6.0
   Compiling yaml-rust v0.4.4
   Compiling textwrap v0.11.0
   Compiling dlib v0.4.2
   Compiling freetype-sys v0.13.1
   Compiling expat-sys v2.1.6
   Compiling phf v0.8.0
   Compiling rusttype v0.9.2
   Compiling quote v1.0.7
   Compiling gl_generator v0.14.0
   Compiling deflate v0.8.6
   Compiling regex-automata v0.1.9
   Compiling net2 v0.2.35
   Compiling iovec v0.1.4
   Compiling memmap v0.7.0
   Compiling parking_lot_core v0.8.0
   Compiling dirs-sys v0.3.5
   Compiling inotify-sys v0.1.3
   Compiling signal-hook-registry v1.2.1
   Compiling raw-window-handle v0.3.3
   Compiling shared_library v0.1.9
   Compiling filetime v0.2.12
   Compiling atty v0.2.14
   Compiling time v0.1.44
   Compiling andrew v0.3.1
   Compiling xcb v0.9.0
   Compiling wayland-scanner v0.28.2
   Compiling vte_generate_state_changes v0.1.1 (https://github.com/alacritty/vte?rev=4f44023dab081f7da74fee14bc53b10ee8f96a1e#4f44023d)
error: failed to run custom build command for `expat-sys v2.1.6`

Caused by:
  process didn't exit successfully: `/home/meneal/Documents/gits/external/alacritty/target/release/build/expat-sys-4c7733d7813e1324/build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-env-changed=EXPAT_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=EXPAT_STATIC
  cargo:rerun-if-env-changed=EXPAT_DYNAMIC
  cargo:rerun-if-env-changed=PKG_CONFIG_ALL_STATIC
  cargo:rerun-if-env-changed=PKG_CONFIG_ALL_DYNAMIC
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  running: "cmake" "/home/meneal/.cargo/registry/src/github.com-1ecc6299db9ec823/expat-sys-2.1.6/expat" "-DBUILD_shared=OFF" "-DBUILD_tools=OFF" "-DBUILD_examples=OFF" "-DBUILD_tests=OFF" "-DCMAKE_INSTALL_PREFIX=/home/meneal/Documents/gits/external/alacritty/target/release/build/expat-sys-057eed882b41d6dc/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_CXX_COMPILER=c++" "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_ASM_COMPILER=/usr/bin/cc" "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
  -- The C compiler identification is GNU 10.2.1
  -- The CXX compiler identification is unknown
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: /usr/bin/cc - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- Configuring incomplete, errors occurred!
  See also "/home/meneal/Documents/gits/external/alacritty/target/release/build/expat-sys-057eed882b41d6dc/out/build/CMakeFiles/CMakeOutput.log".
  See also "/home/meneal/Documents/gits/external/alacritty/target/release/build/expat-sys-057eed882b41d6dc/out/build/CMakeFiles/CMakeError.log".

  --- stderr
  CMake Error at CMakeLists.txt:4 (project):
    The CMAKE_CXX_COMPILER:

      c++

    is not a full path and was not found in the PATH.

    Tell CMake where to find the compiler by setting either the environment
    variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
    to the compiler, or to the compiler name if it is in the PATH.


  thread 'main' panicked at '
  command did not execute successfully, got: exit code: 1

  build script failed, must exit now', /home/meneal/.cargo/registry/src/github.com-1ecc6299db9ec823/cmake-0.1.44/src/lib.rs:885:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
```

### Diagnostic

`g++` is not one of the dependencies, but with it installed there are no errors.

### System

OS: Linux (Fedora 33)
Version: building from master branch
Linux/BSD:  Wayland, Gnome

### Logs

Crashes: STDERR, STDOUT
Font/Terminal size: `alacritty -vv`
Keyboard and bindings: `alacritty --print-events`

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -113,7 +113,7 @@ command that should install all of them. If something is still found to be
 missing, please open an issue.
 
 ```sh
-dnf install cmake freetype-devel fontconfig-devel libxcb-devel
+dnf install cmake freetype-devel fontconfig-devel libxcb-devel g++
 ```
 
 #### CentOS/RHEL 7

